sources/packages/backend/apps/api/src/route-controllers/student-appeal/_tests_/e2e/student-appeal.aest.controller.approveStudentAppealRequests.e2e-spec.ts

```diff
@@ -74,31 +88,166 @@ describe("StudentAppealAESTController(e2e)-approveStudentAppealRequests", () =>
           id: true,
           appealStatus: true,
           assessedBy: { id: true },
-          note: { description: true },
+          note: { description: true, noteType: true },
         },
+        studentAssessment: { id: true, triggerType: true },
+      },
+      relations: {
+        appealRequests: { assessedBy: true, note: true },
+        studentAssessment: true,
       },
-      relations: { appealRequests: { assessedBy: true, note: true } },
       where: { id: appeal.id },
     });
     // Approving ministry user.
     const ministryUser = await getAESTUser(
       db.dataSource,
       AESTGroups.BusinessAdministrators,
     );
-    // Asset updated appeal and appeal request.
+    // Assert updated appeal and appeal request.
     expect(updatedAppeal).toEqual({
       id: appeal.id,
+      studentAssessment: {
+        id: expect.any(Number),
+        triggerType: AssessmentTriggerType.StudentAppeal,
+      },
       appealRequests: [
         {
           id: savedAppealRequest.id,
           appealStatus: StudentAppealStatus.Approved,
           assessedBy: { id: ministryUser.id },
-          note: { description: "Approved" },
+          note: { description: "Approved", noteType: NoteType.Application },
         },
       ],
     });
   });
 
+  describe("Modified independent appeal", () => {
+    for (const modifiedIndependentStatus of [
+      ModifiedIndependentStatus.Approved,
+      ModifiedIndependentStatus.Declined,
+    ]) {
+      it(`Should update the student profile modified independent status as ${modifiedIndependentStatus} when a valid modified independent appeal is assessed.`, async () => {
+        // Arrange
+        // Create a completed application to request change.
+        const student = await saveFakeStudent(db.dataSource);
+        // Create pending student appeal.
+        const appealRequest = createFakeStudentAppealRequest(undefined, {
+          initialValues: {
+            appealStatus: StudentAppealStatus.Pending,
+            submittedFormName: MODIFIED_INDEPENDENT_APPEAL,
+            submittedData: {
+              actions: [StudentAppealActionType.UpdateModifiedIndependent],
+            },
+          },
+        });
+        const appeal = createFakeStudentAppeal({
+          student,
+          appealRequests: [appealRequest],
+        });
+        await db.studentAppeal.save(appeal);
+        const [savedAppealRequest] = appeal.appealRequests;
```

Comment:
Minor. Do we require a new variable for appealRequest as `appealRequest ` is there?

Reply:
I will switch it.

Reply:
Done.